### DISCUSSION TITLE
Mint Layer

### DISCUSSION THREAD

**Discussion by jgarte** [syl20bnr/spacemacs#14798] [Ideas]
Hi, I wanted to open up discussion about adding a [mint](https://www.mint-lang.com/) layer to spacemacs:

Here are some potential pieces to the layer:

https://github.com/creatorrr/emacs-mint-mode

## Language Server Protocol

The language server for mint is built into the language itself. 

See here:

https://github.com/mint-lang/mint/tree/df3ac8b3ef81a2f3a5e46ccc09d8a1b3cd16066e/src/ls

and here:

https://spectrum.chat/mint-lang/development/language-server-support~227be091-fb65-40fb-ab60-952be791b28d

**Comment by lebensterben** [comment]
This mode is 3 years old. Is it still under active development?
